### DISCUSSION TITLE
当音频内容较多时，利用whisper转录后，有些句子的开头几个音节可能丢失，能否提供一种方式自己来替换这个一段segment，因为重新生成可能其他地方也有问题，得不偿失

### DISCUSSION THREAD

**Discussion by imxw** [ZuodaoTech/everyone-can-use-english#921] [Ideas]
[Breakfast(144.35s-149.11s).mp3.zip](https://github.com/user-attachments/files/16454330/Breakfast.144.35s-149.11s.mp3.zip)
比如这段前面的内容直接没了
<img width="486" alt="image" src="https://github.com/user-attachments/assets/e2ad522d-95d3-4e6e-b095-005a0bb12f6b">

**Comment by an-lee** [comment]
点字幕列表右上角的按钮，可以修改字幕，时间轴和文本都可以修改。

**Comment by an-lee** [comment]
播放器上点剪刀按钮，也可以修改 segment 的时间区间。
